select_command/_index.md

```diff
@@ -0,0 +1,60 @@
+
+# <center><b>Mission 1: Save the Planet of Fun From Boredom!</b></center>

```

Comment:
The `_index.md` page should be an introduction to this workshop - I think the content here should be moved to an `activity1.md` page.

Reply:
I see, I renamed the file activity1.md to reflect the changes. Thanks Oliver!